start.py

```diff
@@ -46,7 +46,7 @@ def save_config_and_exit():
         code_name = sys.argv[1]
     else:
         code_name = None
-        while (code_name == None or any([x in code_name for x in ["-", "=", ".", " "]])):
+        while (code_name == None or any([x not in "qwertyuiopasdfghjklzxcvbnm" for x in code_name])):
```

Comment:
Slightly odd choice to use the QWERTY layout to organise the alphabet; I'd say it obscures the intent a bit and makes it harder to spot at a glance if any letters have been missed, though since all 26 letters are indeed there, it will of course do the trick anyway.